getting_hired/preparing_for_job_search/project_portfolio.md

```diff
@@ -13,6 +13,7 @@ Don't underestimate the importance of high quality documentation on your project
 Remember to write some documentation for both technical and non-technical audience.  A potential employer should be able to follow your documentation, regardless of their technical background.
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
+* Looking for a way to make your portfolio standout? Consider participating in open source projects. Check out this [Dev.to article](https://dev.to/opensauced/how-i-got-hired-contributing-to-open-source-projects-546i) to learn how joining the open source community can aid you in your job search.
```

Comment:
Thanks for adding your explanation, but this information does not fit under the "Documentation" heading in this project lesson. I think the lesson that you indicated on the issue will be a much better fit.

In addition, since TOP is open source, I really think we should also mention that they can contribute to our lessons, our main site, odin-bot-v2, etc.

Reply:
Hi @rlmoser99 I made the change! :)